### PR TITLE
[DOCS] Popup.popup_exclusive description

### PR DESCRIPTION
**Pull request by ghost**
Self-explanatory

### CODE REVIEW

File: doc/classes/Popup.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
 	</methods>
 	<members>
 		<member name="popup_exclusive" type="bool" setter="set_exclusive" getter="is_exclusive">
+			If [code]true[/code], the popup will not be hidden when a click event occurs outside of it, or when it receives the [code]"ui_cancel"[/code] action event.
```
Comment (mhilbrunner) [inline doc/classes/Popup.xml]: Remove comma after `If [code]true[/code]`

Remove quotes in `[code]"ui_cancel"[/code]`

Comment (homer666) [reply doc/classes/Popup.xml]: Done.

**Comment by mhilbrunner** [comment]
Thanks a bunch!

**Comment by hpvb** [comment]
Cherry picked into 3.0.3

**Pull request merged**